### PR TITLE
add cloudbees-developers to second tier

### PR DESCRIPTION
**Pull request by car-roll**
# Description
This PR adds the team `cloudbees-developers` to another possible set of cloudbees-owned plugins.
Keeping in draft right now as this is a first pass.

# Submitter checklist for adding or changing permissions

### Always

- [ ] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by jglick** [comment]
compare #2265

File: permissions/component-pipeline-model-json-shaded.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ github: "jenkinsci/pipeline-model-definition-plugin"
 paths:
 - "org/jenkins-ci/lib/pipeline-model-json-shaded"
 developers:
+- "@cloudbees-developers"
```
Comment (jglick) [inline permissions/component-pipeline-model-json-shaded.yml:7]: Does this still exist?

File: permissions/plugin-blueocean-autofavorite.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/blueocean-autofavorite"
 developers:
+- "@cloudbees-developers"
```
Comment (jglick) [inline permissions/plugin-blueocean-autofavorite.yml:9]: I hope only to deprecate it!

File: permissions/plugin-blueocean-executor-info.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ issues:
 paths:
 - "io/jenkins/blueocean/blueocean-executor-info"
 developers:
+- "@cloudbees-developers"
```
Comment (jglick) [inline permissions/plugin-blueocean-executor-info.yml:9]: IIRC this was already deleted?

File: permissions/plugin-pipeline-model-declarative-agent.yml [hunk 4]
```diff
@@ -6,6 +6,7 @@ issues:
 paths:
 - "org/jenkinsci/plugins/pipeline-model-declarative-agent"
 developers:
+- "@cloudbees-developers"
```
Comment (jglick) [inline permissions/plugin-pipeline-model-declarative-agent.yml:9]: Was deleted years ago.

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Same as #2265

File: permissions/plugin-caffeine-api.yml [hunk 5]
```diff
@@ -4,6 +4,7 @@ github: &GH "jenkinsci/caffeine-api-plugin"
 paths:
 - "io/jenkins/plugins/caffeine-api"
 developers:
+- "@cloudbees-developers"
```
Comment (jtnord) [inline permissions/plugin-caffeine-api.yml:7]: +1

**Comment by car-roll** [comment]
obsolete PR, closing

**Pull request closed without merging**